### PR TITLE
Really fast forward PoC of a package manager.

### PR DESCRIPTION
**Pull request by Natim**
Just for the demo.

### CODE REVIEW

File: ideascube/serveradmin/views.py [hunk 1]
```diff
@@ -167,3 +170,41 @@ def wifi_history(request):
...
+    packages = client.get_records()
+
+    return render(
+        request, 'serveradmin/packages.html',
+        {'packages': packages})
+
+
+@staff_member_required
+def package_detail(request, package_id):
+    client = Client(server_url='http://kinto.bsf-intranet.org/v1',
```
Comment (almet) [inline ideascube/serveradmin/views.py]: the url should come from the configuration.

Comment (almet) [inline ideascube/serveradmin/views.py]: The server might not have access to an external repository here. Thhis brings a use case of an offline-first client for Kinto.py (which is good) but also means that this PoC doesn't handle the offline case just yet.

Comment (Natim) [inline ideascube/serveradmin/views.py]: Note that you cannot install a remote package while offline, so this is a limitation of the feature.
But yeah, it definitely brings the usecase for offline first kinto.py :) I opened  about this https://github.com/Kinto/kinto.py/issues/25

Comment (almet) [inline ideascube/serveradmin/views.py]: You could at least list them so someone can select them without any connection and then create a script (or anything) that retrieves the packages to put on an USB stick, for instance.

**Comment by yohanboniface** [comment]
#281 ;)

**Pull request closed without merging**